frontend/src/ui-components/AcmTable/AcmTable.tsx

```diff
@@ -1162,8 +1217,45 @@ export function AcmTable<T>(props: AcmTableProps<T>) {
               <AcmManageColumn<T>
                 {...{ selectedColIds, setSelectedColIds, requiredColIds, defaultColIds, setColOrderIds, colOrderIds }}
                 allCols={columns.filter((col) => !col.isActionCol)}
+                toolbarItemSpacerSetting={showExportButton ? { default: 'spacerNone' } : undefined}
```

Comment:
I think a more elegant solution than conditionally adding space to the right of this item would be to put the managed columns button and export dropdown in a group of their own:
```
<ToolbarGroup spaceItems={{ default: 'spaceItemsNone' }}>
```

Reply:
Excellent solution, thank you.